packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -36,7 +36,7 @@ class RenderParagraph extends RenderBox {
   // TODO(abarth): This logic should live in TextPainter and be shared with RenderEditableLine.
   void _layoutText(BoxConstraints constraints) {
     assert(constraints != null);
-    assert(constraints.debugAssertIsNormalized);
+    assert(constraints.debugAssertIsValid());
```

Comment:
Ok. I won't bother adding the extra assert here then since it would just add extra places for people to trip over that aren't actually real bugs.